library/src/main/java/com/github/pwittchen/reactivenetwork/library/network/observing/strategy/PreLollipopNetworkObservingStrategy.java

```diff
@@ -51,7 +51,12 @@
 
         subscriber.add(unsubscribeInUiThread(new Action0() {
           @Override public void call() {
-            context.unregisterReceiver(receiver);
+            //Catch case where the receiver is already unregistered
+            try {
+              context.unregisterReceiver(receiver);
+            } catch (Exception e) {
+              // ignore exception
```

Comment:
The thing is it is safe to ignore the exception because if we want to unregister the receiver and it's not registered, then by definition it is unregistered anyway, no?